ui/shared/components/panel/variants/FamilyVariantTags.jsx

```diff
@@ -205,124 +216,150 @@ const VariantNoteField = ({ action, note, variant, family, ...props }) => {
 
 VariantNoteField.propTypes = {
   note: PropTypes.object,
-  variant: PropTypes.object,
+  variant: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
   action: PropTypes.string,
   family: PropTypes.object.isRequired,
 }
 
+const VariantLink = (
+  { variant, savedVariant, family },
+) =>
+  <VariantLinkContainer>
+    <NavLink
+      to={savedVariant ?
+        `/project/${family.projectGuid}/saved_variants/variant/${savedVariant.variantGuid}` :
+        `/variant_search/variant/${variant.variantId}/family/${family.familyGuid}`
+      }
+      activeStyle={NO_DISPLAY}
+    >
+      <Popup
+        trigger={<Icon name="linkify" link />}
+        content="Go to the page for this individual variant. Note: There is no additional information on this page, it is intended for sharing specific variants."
+        position="right center"
+        wide
+      />
+    </NavLink>
+  </VariantLinkContainer>
+
+VariantLink.propTypes = {
+  variant: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+  savedVariant: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+  family: PropTypes.object,
+}
+
 const FamilyVariantTags = (
-  { variant, savedVariant, family, project, dispatchUpdateVariantNote, dispatchUpdateFamilyVariantTags },
-) => (
-  family ?
-    <div>
-      <InlineContainer>
-        <InlineHeader size="small">
-          Family<HorizontalSpacer width={5} />
-          <PopupWithModal
-            hoverable
-            style={FAMILY_POPUP_STYLE}
-            position="right center"
-            keepInViewPort
-            trigger={
-              <ColoredLink
-                to={`/project/${family.projectGuid}/family_page/${family.familyGuid}`}
-                color={FAMILY_ANALYSIS_STATUS_LOOKUP[family[FAMILY_FIELD_ANALYSIS_STATUS]].color}
-              >
-                {family.displayName}
-              </ColoredLink>
-            }
-            content={<Family family={family} fields={FAMILY_FIELDS} useFullWidth disablePedigreeZoom />}
-          />
-        </InlineHeader>
-      </InlineContainer>
-      <InlineContainer>
-        <div>
-          <TagTitle>Tags:</TagTitle>
-          <HorizontalSpacer width={5} />
-          <ShortcutTags variant={savedVariant || variant} familyGuid={family.familyGuid} dispatchUpdateFamilyVariantTags={dispatchUpdateFamilyVariantTags} />
-          <VariantTagField
-            field="tags"
-            fieldName="Tags"
-            family={family}
-            variant={savedVariant || variant}
-            tagOptions={project.variantTagTypes.filter(vtt => vtt.name !== NOTE_TAG_NAME)}
-            onSubmit={dispatchUpdateFamilyVariantTags}
-          />
-          <HorizontalSpacer width={5} />
-          {savedVariant && savedVariant.tags.some(tag => tag.category === DISCOVERY_CATEGORY_NAME) &&
+  { variant, savedVariant, family, project, dispatchUpdateVariantNote, dispatchUpdateFamilyVariantTags, isCompoundHet, areCompoundHets },
```

Comment:
what is the difference between `isCompoundHet` and `areCompoundHets`?

Reply:
`isCompoundHet` is for checking if a single variant is compound het, while `areCompoundHets` is for checking if a list of variants are compound hets (to style the main gene section).